lib/features/thread/presentation/thread_bindings.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:core/core.dart';
```

Comment:
A lot of format change, please remove it

Reply:
Removed all of the format changes.